### PR TITLE
created booking index, booking create, booking destroy, offer create …

### PR DESCRIPTION
**Pull request by tuoanhnt95**
…in sdashboard

### CODE REVIEW

File: app/controllers/offers_controller.rb [hunk 1]
```diff
@@ -11,4 +11,29 @@ def show
...
+  def create
+    @offer = Offer.new(offer_params)
+    @offer.user = current_user
+    authorize @offer
+    if @offer.save
+      redirect_to dashboard_path
+    else
+      # render 'new.html.erb'
+      # redirect_to new_restaurant_path
+      render :new
```
Comment (SummerSoong) [inline app/controllers/offers_controller.rb:29]: , status: :unprocessable_entity

Comment (mengrui-song) [inline app/controllers/offers_controller.rb]: render :new, status: :unprocessable_entity

File: app/controllers/pages_controller.rb [hunk 2]
```diff
@@ -3,8 +3,9 @@ def home
   end
 
   def dashboard
```
Comment (mengrui-song) [inline app/controllers/pages_controller.rb:5]: def dashboard
    skip_authorization
    # I booked
    @make_bookings = current_user.bookings

    # I am booked by => offers I made, and booked by someone
    @booked_by = current_user.bookings_as_owner
    
    # all offers I made
    current_user.offers
  end

Comment (mengrui-song) [reply app/controllers/pages_controller.rb:5]: We also need some has_many...relationship in models

**Review by mengrui-song — APPROVED** [review]
I will fix the model relationship and page#dashboard

**Pull request merged**